newIDE/app/src/AssetStore/BehaviorStore/BehaviorStoreContext.js

```diff
@@ -25,6 +25,10 @@ export type SearchableBehaviorMetadata = {|
   fullName: string,
   description: string,
   objectType: string,
+  /**
+   * All required behaviors including transitive ones.
```

Comment:
Is the meaning of "transitive" documented somewhere?

Reply:
It's a mathematical term, but commonly used I guess. I meant required of required... of required behaviors.
Maybe this phrasing is more accurate: `All required behaviors including the ones required transitively.`?

Reply:
I think it could be explained somewhere. I understand the meaning of transitive, but transitive through what path? required behaviors of the required behaviors? Between event based objects and this logic, sometimes it's a bit hard to understand what is implied when you're not used to work on this part of the codebase